cli/core/test/e2e/plugin_lifecycle/plugin_lifecycle_test.go

```diff
@@ -0,0 +1,41 @@
+// plugin_e2e_tests provides plugin command specific E2E test cases
+package plugin_e2e_tests
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/vmware-tanzu/tanzu-framework/cli/core/test/e2e/framework"
+)
+
+var _ = framework.CLICoreDescribe("[Tests:E2E][Feature:Plugin-lifecycle]", func() {
+	var (
+		tf        *framework.Framework
+		pluginOps framework.PluginOps
+	)
+	BeforeEach(func() {
+		tf = framework.NewFramework()
+		pluginOps = framework.NewLocalOCIPluginOps(framework.NewLocalOCIRegistry(framework.DefaultRegistryName, framework.DefaultRegistryPort))
+	})
+	Context("oci local registry - plugin life cycle basic operations", func() {
```

Comment:
May be something like
```suggestion
	Context("OCI repository based plugin lifecycle operations tests", func() {
```

Reply:
done